docs/UserGuide.md

```diff
@@ -155,18 +155,42 @@ Examples:
 
 Loans a book to some user, which has a due date.
 
-Format: `loan USER_INDEX BOOK_INDEX [DUE DATE]`
+Format: `loan USER_INDEX BOOK_INDEX [DUE_DATE]`
 
 * Loans the book to some user at their respective specified `INDEXES`.
 * The indexes refer to the index number shown in the displayed user and book list respectively.
 * The indexes **must be a positive integer** 1, 2, 3, …​
 * The respective specified `INDEXES` **must be present in their lists**.
-* The books that are loaned out will appear at the top of the book list.
+* The books that are loaned out will appear at the top of the book list after closing and reopening of app.
 * If due date is not specified, a default due date of 14 days from today is set when the book is loaned out.
-* Due date formats such as `dd/MM/yyyy`, `yyyy-MM-dd` or even text such as `next sunday` or `tomorrow` would work. Only the
-  first date entered would be set as the due date and subsequent dates entered would be ignored.
+* Due date formats such as `dd/MM/yyyy`, `yyyy-MM-dd` or even text such as `next sunday`, `tomorrow` or `two mondays ago` would work.
+* Compound/complex statements or other date formats not stated below may not work properly.  (Fix coming in v1.5)
+Eg. `one month and two weeks later` or `2022-10-31 next sunday` will not work properly.
 * Some invalid due date inputs in February may be assumed to be correct. Refer to example below.
 
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+{DAY} refers to Sunday, Monday...Saturday or day(s)/week(s)/month(s)/year(s)
+
+Date formats that will work:
+dd/MM/yyyy, yyyy-MM-dd, next {DAY}, following {DAY}, last {DAY}, past {DAY}, yesterday, today, tomorrow, (any positive integer) {DAY} later,
+(any positive integer) {DAY} ago.
+</div>
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Notes about the command format:**<br>
+
+* Next strictly refers to next week while following refers to the upcoming weekday/weekend.
```

Comment:
Do we wanna add a comparison between `this {DAY}` and `next {DAY}`, given that users expect `this {DAY}` to return today's date if today is the `{DAY}`

Reply:
`this {DAY}` is not stated in the command format which means it isnt supported, so if users enter `this {DAY}` then we can claim that its coming in the future and is out of scope. 

In addition there's no real reason to use `this {DAY}` if today is the `{DAY}`, users will simply use `today`

Reply:
Ah yea, so it falls under compound/complex then